### ISSUE TITLE
Use liblouis 3.0.0 (incl. its nl-NL table)

### ISSUE THREAD

**Issue by dkager** [snaekobbi/pipeline-mod-dedicon#50]
@bertfrees Any hints?

**Comment by bertfrees** [comment]
pipeline-mod-braille 1.9.15 uses liblouis 3.0.0-alpha, but not the new tables yet. So two things need to happen:
- [ ] if 3.0.0-alpha won't do, update to 3.0.0 (this means publishing a new liblouis-nar and updating the liblouis-native module)
- [ ] update the tables (this means copying them to the liblouis-tables module)

Is your table in mod-dedicon identical to the one in liblouis?

**Comment by dkager** [comment]
> Is your table in mod-dedicon identical to the one in liblouis?

No, it is a backport to 2.6.5 and therefore avoids using some opcodes.

I'm assuming @egli has to publish the final 3.0.0 and then it can be included in pipeline-mod-braille. I think we can wait for that to happen. Presumably this also means moving the nl-NL table from mod-dedicon to the tables module.

**Comment by bertfrees** [comment]
Ah yes. But the current table can safely be replaced by the new one in Liblouis 3.0.0?

Christian has already made the final 3.0.0 AFAIK.

We don't need to move anything to the tables module. We just update the liblouis-tables module to 3.0.0, and then remove the table from mod-dedicon and point to the new one from liblouis-tables instead.

**Comment by dkager** [comment]
> But the current table can safely be replaced by the new one in Liblouis 3.0.0?

Yes. All we keep in mod-dedicon is the undefined character one-liner I think.

> Christian has already made the final 3.0.0 AFAIK.

But not anything with liblouis-nar if I read your earlier comment correctly.

**Comment by bertfrees** [comment]
I need to do that anyway because of the Mac binaries, Christian can't build those on his machine.

**Comment by bertfrees** [comment]
@egli It seems that if you do a fresh checkout of liblouis v3.0.0, `./autogen.sh` gives you an error because the m4 directory isn't there. You did something in commit https://github.com/liblouis/liblouis/commit/60e5bf83fe9bb1aa6d731eb872c710e9a90b4fe0. Maybe add a m4/.gitkeep file?

**Comment by bertfrees** [comment]
Or just add a `mkdir m4` to autogen.sh?

**Comment by bertfrees** [comment]
Currently stuck on https://github.com/liblouis/liblouis-java/commit/5542a87bd749e5174e9b1d2c358b51794cd37449.

**Comment by egli** [comment]
Don't know if I ever removed the `m4` directory. Will go for the `m4/.gitkeep` solution
